### ISSUE TITLE
`correct_images process .` does not run unless `-F` flag is set

### ISSUE THREAD

**Issue by adrian-bodenmann** [ocean-perception/oplab_pipeline#66]
When running 
`correct_images parse .`
followed by 
`correct_images process .`
on a (new) dataset, correct_images process aborts with the error message
```
**** Exiting.
**** Reason: Parameters exist for current configuration. Run parse with Force (-F flag)...
```

**Comment by adrian-bodenmann** [comment]
Duplicate of https://github.com/ocean-perception/oplab_pipeline/issues/50

**Comment by adrian-bodenmann** [comment]
Addressed in 6682f6466b82a8d77a9074c490bf487a56c85e35
